components/ide/jetbrains/gateway-plugin/gradle-latest.properties

```diff
@@ -1,10 +1,10 @@
 # See https://plugins.jetbrains.com/docs/intellij/build-number-ranges.html
 # for insight into build numbers and IntelliJ Platform versions.
-pluginSinceBuild=233.9102
+pluginSinceBuild=233.8264
```

Comment:
ignore this PR, it is downgrade, it will be auto resolved then JB publish EAP this week